packages/html2react/src/index.ts

```diff
@@ -1,12 +1,13 @@
 import Html2React from "../types";
 import Component from "./libraries/component";
 import script from "../processors/script";
+import link from "../processors/link";
 
 const html2react: Html2React = {
   name: "@frontity/html2react",
   libraries: {
     html2react: {
-      processors: [script],
+      processors: [script, link],
```

Comment:
I guess we have to make a decision here: should the `link` processor be added by default or should be added by the theme creator like the `image` and `iframe` processors?

My opinion now is that we should leave it out for the moment, as the `<Link>` component itself is optional in the themes. But we should improve the way people share processors. The conversation is happening here: https://community.frontity.org/t/how-to-share-html2react-processors/3069. It would be awesome if you could give us your feedback @nicholasio 🙂.

The best idea we had so far is to make them similar to the way `fills` work now. It is a bit verbose, but it is consistent and very flexible.

`script` is not optional because if not the `<script>` tags of the `post.content` don't work anymore.

Reply:
Makes sense to leave it out for now - but we need to make this very clear in the documentation as this is definitely going to be a must-need feature for most themes.

Reply:
I was going to say that it makes sense but if the script is not optional it means that all themes are already shipping with frontity/components package, so I don't see a reason why not to make the link processor not be enabled by default.

Reply:
Actually, I see your point, you're concerned more about the way processors are shared and not themes needing to declare `frontity/components` as a dependency. I'll make it optional and include in the default themes.

Reply:
Done.

Reply:
Thanks @nicholasio :)